src/dmd/backend/mscoffobj.d

```diff
@@ -2077,16 +2077,12 @@ static if (0)
     int save = cast(int)buf.size();
     //dbg_printf("MsCoffObj_bytes(seg=%d, offset=x%lx, nbytes=%d, p=x%x)\n",
             //seg,offset,nbytes,p);
-    buf.setsize(cast(uint)offset);
-    buf.reserve(nbytes);
+    buf.position(cast(size_t)offset, nbytes);
```

Comment:
Not that It particularly matters or this PR, but this whole function `MsCoffObj_bytes ` appears to be an exact duplicate of `Obj_bytes` in both `machobj.d` and `elfobj.d`

Reply:
Yep! The backend has a lot of duplications like this... I'm trying to get rid of *some* of it to make my issues easier to fix.